spec/active_hash/base_spec.rb

```diff
@@ -404,6 +404,19 @@ class Region < ActiveHash::Base
     end
   end
 
+  describe ".pluck" do
+    before do
+      Country.data = [
+        {:id => 1, :name => "US"},
+        {:id => 2, :name => "Canada"}
+      ]
+    end
+
+    it "returns an Array of attribute values" do
+      expect(Country.pluck(:id, :name)).to match_array([[1,"US"], [2, "Canada"]])
+    end
```

Comment:
if calling `#pluck` with single symbol(`Country.pluck(:id)`),  it returns flatten array. right?(not like `[[1],[2]]`)
so please add the case